js/navigation/styles.js

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from "react-native";
+export default (style = StyleSheet.create({
+  header: {
+    flex: 1,
+    borderBottomColor: "white"
+  },
+  profileIcon: {
+    height: 35,
+    width: 35,
+    alignSelf: "flex-end",
+    marginRight: 10
+  }
```

Comment:
this is not how you export your styles. Make a const name styles and then export default that const at the end of your file.